src/misc.jl

```diff
@@ -395,17 +351,17 @@ end
 discretize_make_ia(values::AbstractVector, levels) =
     IndirectArray(Array{UInt8}(indexin(values, levels)), levels)
```

Comment:
@nalimilan Shouldn't the last parameter be `values` not `levels`? 

```
IndirectArray(Array{UInt8}(indexin(values, levels)), values)
```

Also when playing around with this I think I ran into a massive bug in IndirectArrays: https://github.com/JuliaArrays/IndirectArrays.jl/issues/18